integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/LookupPO.java

```diff
@@ -0,0 +1,53 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import to.etc.domui.webdriver.core.WebDriverConnector;
+
+@NonNullByDefault
+public class LookupPO extends BasePO {
+
+	private final String m_id;
+
+	public LookupPO(WebDriverConnector connector, String id) {
+		super(connector);
+		m_id = id;
+	}
+
+	public void setValue(String val, String colName) {
```

Comment:
As discussed, we need to change these. We can do that later, but it needs to be done right after to prevent fragile tests.